awx/main/migrations/0133_unifiedjob_ansible_version.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.16 on 2021-03-16 20:49
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0132_instancegroup_is_container_group'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='unifiedjob',
+            name='ansible_version',
+            field=models.CharField(blank=True, default='', editable=False, help_text='The version of Ansible Core installed in the execution environment.', max_length=50),
```

Comment:
@beeankha the `50` in this migration should be updated to match too

Reply:
Oop, sorry to miss that! Fixed!